src/cloud/components/onboarding/useGetUserStatus.ts

```diff
@@ -0,0 +1,110 @@
+import {useEffect, useState, useCallback, useMemo} from 'react'
+import {getOrgsUsage} from 'src/client'
+import {fromFlux} from '@influxdata/giraffe'
+import {usageStatsCsv} from 'src/shared/utils/mocks/usagestats'
+import {useSelector} from 'react-redux'
+import {getOrg} from 'src/organizations/selectors'
+import {event} from 'src/cloud/utils/reporting'
+import {isFlagEnabled} from 'src/shared/utils/featureFlag'
+
+export enum USER_PILOT_USER_STATUS {
+  NEW_USER = 'NEW_USER',
+  DELETED_ALL_DATA = 'DELETED_ALL_DATA',
+  NON_WRITING_USER = 'NON_WRITING_USER',
+  ACTIVE_USER = 'ACTIVE_USER',
+  ACTIVE_WRITING_USER = 'ACTIVE_WRITING_USER',
+  WRITING_NOT_READING_USER = 'WRITING_NOT_READING_USER',
+}
+export const getUserStatus = (table: any): USER_PILOT_USER_STATUS[] => {
```

Comment:
Let’s set the table type here

Reply:
ah sorry about that, forgot that one, any was temporary and forgot to change back. Thanks for catching